x/concentrated-liquidity/store_test.go

```diff
@@ -75,3 +76,51 @@ func (s *KeeperTestSuite) TestGetAllPositionsWithVaryingFreezeTimes() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestParseFullPositionFromBytes() {
+	defaultFrozenUntil := s.Ctx.BlockTime().Add(DefaultFreezeDuration)
+	defaultAddress := s.TestAccs[0]
+	cdc := s.App.AppCodec()
+
+	tests := map[string]struct {
+		key          []byte
+		val          []byte
+		expectingErr bool
+	}{
+		"empty val": {
+			key:          types.KeyFullPosition(defaultPoolId, defaultAddress, DefaultLowerTick, DefaultUpperTick, defaultFrozenUntil),
+			val:          []byte{},
+			expectingErr: true,
+		},
+		"empty key": {
+			key:          []byte{},
+			val:          cdc.MustMarshal(&model.Position{Liquidity: DefaultLiquidityAmt, FrozenUntil: defaultFrozenUntil}),
+			expectingErr: true,
+		},
+		"sufficient test case": {
```

Comment:
Please also add tests where we format incorrect values for each of the fields. For example, pool id is not uint but a sequence of characters. As a result, it fails to be parsed

Reply:
sounds sense